pkg/kubectl/cmd/create_configmap.go

```diff
@@ -55,7 +60,7 @@ var (
 // ConfigMap is a command to ease creating ConfigMaps.
 func NewCmdCreateConfigMap(f *cmdutil.Factory, cmdOut io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "configmap NAME [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run]",
+		Use:     "configmap NAME [--from-file=[key=]source] [--from-binary-file=[key=]source] [--from-literal=key1=value1] [--dry-run]",
```

Comment:
What is my fall-back if the data gets miscategorized?  Auto-detection is nice when it works and a pain when it doesn't.  Having default auto-detection makes sense for the common cases as long as it is not impossible to fix the uncommon cases.  Is there a `kubectl set` command `configmaps` and `secrets` that I could use to fix this up?